### PR TITLE
fix(devtools): simplify commitlint and update CODEOWNERS

### PR DESCRIPTION
**Pull request by daniellacosse**

### CODE REVIEW

File: commitlint.config.js
Comment (sbruens) [inline commitlint.config.js:21]: Can we remove storybook too? You're either working on the client, the manager, or if it's the storybook infra I'd argue that's a devtools thing?

**Review by sbruens — COMMENTED** [review]
Big +1 to removing the electron/cordova part. This is much better.

Defer to Vini on the CODEOWNERS changes.

**Review by sbruens — APPROVED** [review]

File: commitlint.config.js
Comment (jyyi1) [reply commitlint.config.js:9]: @sbruens proposed to use comma in #2226, but that somehow broke the other commitlint scopes so it was reverted in #2232

**Pull request merged**
